gateway/src/apicast/policy/local_chain/local_chain.lua

```diff
@@ -38,22 +38,19 @@ end
 -- forward all policy methods to the policy chain
 for _, phase in policy.phases() do
   _M[phase] = function(_, context, ...)
-    local policy_chain = find_policy_chain(context)
+    local policy_chain
+    if phase == 'rewrite' then
```

Comment:
This will break JIT. It will be better to generate different function for the rewrite phase.

Reply:
Right. Fixed :+1: